cpu/cpustress.py

```diff
@@ -83,12 +83,10 @@ def __clear_dmesg():
     @staticmethod
     def __error_check():
         ERROR = None
-        logs = (process.system_output(
-            "dmesg -T --level=alert,crit,err,warn", ignore_status=True, shell=True, sudo=True)).split()
-        for log in logs:
-            for error in errorlog:
-                if error in log:
-                    ERROR = 'Test resulted %s in syslogs"' % error
+        logs = process.system_output("dmesg -Txl 1,2,3,4")
+        for error in errorlog:
+            if error in logs:
+                ERROR = 'Test resulted %s in syslogs"' % error
```

Comment:
@harihare We should ideally be appending the error logs to `ERROR`.

Also, I think it makes more sense to split the logs using `splitlines()` and check for errors, and keep on appending such lines to `ERROR` and finally display them. Please let me know your take on this.
Something like this:
```
ERROR = []
logs = process.system_output("dmesg -Txl 1,2,3,4").splitlines()
for error in errorlog:
    for log in logs:
        if error in log:
            ERROR.append(log)
self.log.debug("\n".join(ERROR))
```

Reply:
I agree with you, I have made the changes accordingly.